INSTALL.md

```diff
@@ -212,6 +215,28 @@ back to the project source-code directory. That way, you can edit the source cod
 without reinstalling the package after a small change.
 * `--single-version-externally-managed` is an option for Python packages instructing the setup tools module to create a package the host's package manager can easily manage.
 
+## Build from Sources with `conda-build`
+
+Intel(R) Extension for Scikit-learn* is easily built from the sources using only one command and `conda-build` utility. 
+
+### Prerequisites for `conda-build`
+
+* any `conda` distribution (`miniforge` is recommended)
+* `conda-build` and `conda-verify` installed in a conda environment
+* (Windows only) Microsoft Visual Studio* 2022
```

Comment:
can we point to general VS/compiler requirements so we would have single place to track/update?

Reply:
Not really in case of conda recipe, vs2022 is default for this repo and included in CI Windows image, vs2019 - for conda-forge feedstock, and build config change is required to set another version and there is no guarantee it will work as expected (but it is unlikely someone will use version other than 2019 or 2022).

I will add mention how to change compiler for conda-build on Win.